backbone/README.md

```diff
@@ -159,6 +160,35 @@ return MyItemView;
 
 **[⬆ back to top](#table-of-contents)**
 
+## Ordering
+
+  - **Outside** `Marionette.View` definition
+
+  0. `dependencies` requirement
+  0. `static` methods
+  0. `config` objects
+
+  - Ordering for `Marionette.View`:
```

Comment:
I'm not sure if I'm following you. Even referencing only the life cycle methods, I've received a specific order when it comes to ordering these methods. While I'm not proposing a strict order I think there is an implicit order anyway.